awscli/customizations/emr/createdefaultroles.py

```diff
@@ -215,14 +215,15 @@ def _construct_role_and_role_policy_structure(
             list.append({'Role': response['Role'], 'RolePolicy': policy})
             return list
 
-    def _check_if_role_exists(self, role_name, parsed_globals):
```

Comment:
Why are we making this public?  If it's not intended to be called from other classes I don't think we should be changing internal vs. public for a bug fix.